src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImplIntTest_getCourtHearings.java

```diff
@@ -0,0 +1,116 @@
+package net.syscon.elite.api.resource.impl;
+
+import net.syscon.elite.api.model.ErrorResponse;
+import net.syscon.elite.executablespecification.steps.AuthTokenHelper;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+
+import java.util.Map;
```

Comment:
For a future PR - there are a couple of unused imports.

Reply:
Nah. I'll sort.  Missed a test rename anyway.